cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -108,8 +108,7 @@
 #}
 
 {% block css %}
-<!--[if lt IE 9]><link rel="stylesheet" href="{{ static('css/main.ie8.css') }}"><![endif]-->
-<!--[if IE 9]><link rel="stylesheet" href="{{ static('css/main.ie9.css') }}"><![endif]-->
```

Comment:
I'm not sure we want to do this. I haven't gone looking for anything in particular, but IE8 and IE9 may have different issues with CSS. Let's keep the scope of this PR to JavaScript.

Reply:
Or at least give me a little time tomorrow to research it further.

Reply:
Differences likely would be fallbacks stacked on top of each other, in which case one file would likely be preferable.